python/setup_project/default_config_page.py

```diff
@@ -56,6 +60,7 @@ def validatePage(self):
             wiz.validate_config_uri(uri)
             wiz.ui.github_errors.setText("")
         except Exception as e:
+            logger.exception("Unexpected error while validating config:")
```

Comment:
I debugged an issue where I didn't have a callstack anywhere, so this became handy.